integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4646,10 +4646,144 @@ msgstr "Seite neu laden"
 msgid "Continue reading"
 msgstr "Weiterlesen"
 
+#: cms/templates/dashboard/_todo_dashboard_widget.html
+msgid "To-Do's"
+msgstr "To-Do’s"
```

Comment:
```suggestion
msgid "To-Do's"
msgstr "To-Dos"
```
To be fair, this was already incorrect in the design :sweat_smile:
https://www.duden.de/rechtschreibung/To_do
http://www.deppenapostroph.info/